STM32F405RGTx/Core/Src/rtos_threads/bist_thread.cpp

```diff
@@ -46,6 +46,12 @@ void BistThread::runBist(void* args) {
         // help
         else if (BistThread::_strcmp(buff, "help"))         { BistThread::_help(); }
 
+        // toggle fault checking
+        else if (BistThread::_strcmp(buff, "toggle_fc"))         { BistThread::_toggle_fc(); }
+
+        // clear
+        else if (BistThread::_strcmp(buff, "clear"))         { BistThread::_clear(); }
+
```

Comment:
Can you get these to line up with the other ones 👀 (The part that's in the curly brackets)